src/plugins/intel_gpu/src/plugin/transformations/transpose_fusion.cpp

```diff
@@ -180,7 +173,7 @@ TransposeMatMulMatcher::TransposeMatMulMatcher() {
     auto matmul_in_a = std::make_shared<Or>(OutputVector{input_a_m, transpose_a_m});
     auto matmul_in_b = std::make_shared<Or>(OutputVector{input_b_m, transpose_b_m});
 
-    auto matmul_m = wrap_type<ov::op::v0::MatMul>({ matmul_in_a, matmul_in_b }, is_dynamic);
+    auto matmul_m = wrap_type<ov::op::v0::MatMul>({ matmul_in_a, matmul_in_b });
```

Comment:
Do I understand correctly, that oneDNN doesn't supports all the transpose combinations (but only listed those in `gemm_inst::is_fusable_permute_input_order_onednn` function)? If so, we may sometime face issues because we don't have any checks for them in these fusion passes for XMX HW?